server/extensions/config/service.go

```diff
@@ -17,10 +17,11 @@ package config
 import (
 	"context"
 
-	forge_types "go.woodpecker-ci.org/woodpecker/v2/server/forge/types"
+	"go.woodpecker-ci.org/woodpecker/v2/server/forge"
+	"go.woodpecker-ci.org/woodpecker/v2/server/forge/types"
 	"go.woodpecker-ci.org/woodpecker/v2/server/model"
 )
 
-type Extension interface {
-	FetchConfig(ctx context.Context, repo *model.Repo, pipeline *model.Pipeline, currentFileMeta []*forge_types.FileMeta, netrc *model.Netrc) (configData []*forge_types.FileMeta, useOld bool, err error)
+type Service interface {
```

Comment:
> Do we call this Service now? Because services are pipeline steps

Right, have to update that as well.

> Also, can you add the new terms (https://github.com/woodpecker-ci/woodpecker/pull/915#issuecomment-1933436736) to the glossary?

Sure, will add it.

Reply:
done